core/src/main/resources/log4j2.properties

```diff
@@ -17,7 +17,7 @@ logger.netty-platformdependent0-nodebug.additivity = false
 appender.console.type = Console
 appender.console.name = STDOUT
 appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = # %highlight{%d{ISO8601} [%p/%c{1}/%t%X{self}] (%F:%L) - %m}{OFF=bright white, FATAL=blink bright red, ERROR=bright red, WARN=bright yellow, INFO=bright white, DEBUG=dim cyan, TRACE=dim white}%n
+appender.console.layout.pattern = %highlight{%d{ISO8601} [%p/%c{1}/%t%X{self}] - %m}{OFF=bright white, FATAL=blink bright red, ERROR=bright red, WARN=bright yellow, INFO=bright white, DEBUG=dim cyan, TRACE=dim white}%n
```

Comment:
I removed the file name and line number to clean it up a bit, I didn't really find it useful. Wdyt?
They're still there in the file appender